assets/Interface/InstructionScreenGUI.xml

```diff
@@ -1,272 +1,272 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
@JohnnyMarek I understand you've created your own `DistractionScreenGUI.xml`, can you revert any changes to "their" file here?